src/applications/mhv/medications/containers/Prescriptions.jsx

```diff
@@ -513,40 +547,33 @@ const Prescriptions = () => {
                     allergiesError,
                 )}
               />
-              <div className="landing-page-content">
-                <PrintDownload
-                  download={handleFullListDownload}
-                  isSuccess={
-                    pdfTxtGenerateStatus.status ===
-                    PDF_TXT_GENERATE_STATUS.Success
-                  }
-                  list
-                />
-                <BeforeYouDownloadDropdown />
-                <MedicationsListSort
-                  value={selectedSortOption}
-                  sortRxList={sortRxList}
-                />
-                <div className="rx-page-total-info vads-u-border-color--gray-lighter" />
-                <MedicationsList
-                  rxList={paginatedPrescriptionsList}
-                  pagination={pagination}
-                  selectedSortOption={selectedSortOption}
-                  updateLoadingStatus={updateLoadingStatus}
-                />
-              </div>
+              {paginatedPrescriptionsList?.length ? (
+                <div className="landing-page-content">
+                  <PrintDownload
+                    download={handleFullListDownload}
+                    isSuccess={
+                      pdfTxtGenerateStatus.status ===
+                      PDF_TXT_GENERATE_STATUS.Success
+                    }
+                    list
+                  />
+                  <BeforeYouDownloadDropdown />
+                  <MedicationsListSort
+                    value={selectedSortOption}
+                    sortRxList={sortRxList}
+                  />
+                  <div className="rx-page-total-info vads-u-border-color--gray-lighter" />
+                  <MedicationsList
+                    rxList={paginatedPrescriptionsList}
+                    pagination={pagination}
+                    selectedSortOption={selectedSortOption}
+                    updateLoadingStatus={updateLoadingStatus}
+                  />
+                </div>
+              ) : (
+                ''
```

Comment:
using empty strings here isn't wrong, but its more semantically correct to use `<></>` or `null` when you don't want to render anything